### PR TITLE
Get started synthetic monitoring

### PR DESCRIPTION
**Pull request by akristen**
Editing content in a top synthetics docs, making it more specific and giving it a refresher. Interested in:

- Feedback on areas that need more specificity
- Places where I can find/put in a use case
- Any place that I can input a gif or screenshot (especially if I can get a gif of what a monitor check looks like?)
- Grammar/copy edits etc
- Suggestions for moving info around, or adding important info upfront

Changes were added based on feedback from #wizards-synthetics. For example, I moved the pricing doc higher in our IA because there's significant enough confusion about how checks fit into data ingest. I figured knowing what you're paying for is important enough to get the coveted Top Section position

### CODE REVIEW

File: src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx [hunk 1]
```diff
@@ -16,32 +16,35 @@ redirects:
...
-* Monitor your API endpoints with [API tests](/docs/synthetics/new-relic-synthetics/scripting-monitors/writing-api-tests).
+Our synthetic monitors are more than just ping monitors. Synthetics can perform scriptable checks on your customer-facing or internal apps, down to the HTTP headers. The goal is to alert you to operational inefficiencies before anyone has a chance to notice.
 
-Ready to get started? If you don't already have one, [sign up for a New Relic account](https://newrelic.com/signup). It's free, forever!
+With synthetic monitoring you can: 
+
+* Expand your browser monitoring with real, Selenium-powered [scripted browsers](/docs/synthetics/new-relic-synthetics/scripting-monitors/writing-synthetic-scripts), which test login procedures, searches, and other critical business transactions.
+* Run an [API test](/docs/synthetics/new-relic-synthetics/scripting-monitors/writing-api-tests) to check certificates, or to send `GET`, `POST`, or `HEAD` requests.
```
Comment (bpeckNR) [inline src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx:30]: We should support any HTTP request type. I worry if we call out GET, POST, and HEAD that customers will think that it is all that is supported. 

We could also call out that additional node modules (https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/using-monitors/manage-monitor-runtimes/#Node16 and https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/scripting-monitors/import-nodejs-modules/#NewRuntime) allow you to expand use case coverage of scripted API monitors.

File: src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx [hunk 2]
```diff
@@ -16,32 +16,35 @@ redirects:
...
-* Monitor your API endpoints with [API tests](/docs/synthetics/new-relic-synthetics/scripting-monitors/writing-api-tests).
+Our synthetic monitors are more than just ping monitors. Synthetics can perform scriptable checks on your customer-facing or internal apps, down to the HTTP headers. The goal is to alert you to operational inefficiencies before anyone has a chance to notice.
 
-Ready to get started? If you don't already have one, [sign up for a New Relic account](https://newrelic.com/signup). It's free, forever!
+With synthetic monitoring you can: 
+
+* Expand your browser monitoring with real, Selenium-powered [scripted browsers](/docs/synthetics/new-relic-synthetics/scripting-monitors/writing-synthetic-scripts), which test login procedures, searches, and other critical business transactions.
+* Run an [API test](/docs/synthetics/new-relic-synthetics/scripting-monitors/writing-api-tests) to check certificates, or to send `GET`, `POST`, or `HEAD` requests. 
+* Automate performance tracking for every deployment by adding synthetic monitoring to your CI/CD pipelines.
```
Comment (bpeckNR) [inline src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx:31]: I would remove this one for now.

File: src/nav/synthetics.yml [hunk 3]
```diff
@@ -7,6 +7,8 @@ pages:
         path: /docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring
       - title: Types of monitors
         path: /docs/synthetics/synthetic-monitoring/getting-started/types-synthetic-monitors
+      - title: Limits on monitors
```
Comment (bpeckNR) [inline src/nav/synthetics.yml:10]: Not really part of this change, but I think it would be less confusing if we changed this to "licensing and usage" or similar instead of "limits on monitors"

File: src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx (hunk 1 above)
Comment (akristen) [reply src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx:30]: @bpeckNR Can you elaborate on what "expand use case coverage" means? It sounds like the new node modules (linked above) expand certain capabilities of the scripted API monitors? Can you be a bit more specific what features were ~added that should be called out?

**Comment by austin-schaefer** [comment]
@akristen : Can you take a look at this PR?

**Comment by akristen** [comment]
@austin-schaefer yes! I'll get this merged this week. Fell through the cracks!

File: src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx [hunk 4]
```diff
@@ -16,32 +16,211 @@ redirects:
...
+  <Collapser
+    className="freq-link"
+    id="compatibility"
+    title="Are there any system requirements with synthetic monitoring?"
+  >
+Synthetic monitoring does not require any software except a [supported browser](/docs/apm/new-relic-apm/getting-started/supported-browsers).
+
+<Callout variant="important">
+  To monitor a site behind your firewall, add the [synthetic monitoring public
+  minion IP
```
Comment (jeff-colucci) [inline src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx:55]: I don't know that this matters but there is some weird formatting here with this link.

Comment (akristen) [reply src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx:55]: LOL. Yes.... this does matter, yes... good catch

File: src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx [hunk 5]
```diff
@@ -16,32 +16,211 @@ redirects:
...
+  to your allow list or [create a private location](/docs/synthetics/synthetic-monitoring/private-locations/private-locations-overview-monitor-internal-sites-add-new-locations/).
+</Callout>
+
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="privacy"
+    title="What kind of data do synthetic monitors create?"
+  >
+The data from synthetic monitoring is test data, representing typical interaction with the webpage or application. It is never actual data from human beings. The data collected when you use synthetic monitoring therefore is not personal data. For more information, see the [Synthetic monitoring security documentation](/docs/synthetics/new-relic-synthetics/getting-started/security-new-relic-synthetics).
```
Comment (jeff-colucci) [inline src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx:66]: When I read this the "therefore" kind of stops my flow. Not a big deal either way, just something to consider.

File: src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx [hunk 6]
```diff
@@ -16,32 +16,211 @@ redirects:
...
+
+    <tr>
+      <td>
+        Simple browser monitors
+
+        API name: `BROWSER`
+      </td>
+
+      <td>
+        Simple browser monitors essentially are simple, pre-built scripted browser monitors. They make a request to your site using an instance of Google Chrome.
```
Comment (jeff-colucci) [inline src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx:178]: Not sure that you need the "essentially" here

File: src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx [hunk 7]
```diff
@@ -16,32 +16,211 @@ redirects:
...
+
+<Video
+  id="WIVOgYlID-E"
+  type="youtube"
+/>
+
+## Advanced testing with scripted browsers [#advanced]
 
-Synthetic monitoring allows you to proactively monitor your website or API endpoint to ensure your content is not only available, but fully functional. Synthetic monitoring browser tests send real, Selenium-powered Google Chrome browsers to your site from locations around the world to ensure your content is always up, everywhere.
+Synthetic monitoring allows you to proactively monitor your website or API endpoint to ensure your content is not only available, but fully functional. [Scripted browsers](/docs/synthetics/new-relic-synthetics/scripting-monitors/writing-synthetic-scripts) send real, Selenium-powered Google Chrome instances to your site from locations around the world, so you're confident that your content is always up, everywhere.
```
Comment (jeff-colucci) [inline src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx:221]: Love the "so you're confident that your content is always up, everywhere" part. I think the comma before "so you're" should be removed.

File: src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx [hunk 8]
```diff
@@ -16,32 +16,211 @@ redirects:
...
 
-[Scripted browsers](/docs/synthetics/new-relic-synthetics/scripting-monitors/writing-synthetic-scripts) expand your testing capabilities, so you can test uncommon user flows or beta-test complex procedures. For example, ensure your users are able to sign up for your newsletter, add an item to their cart, or search for and find a piece of critical content with a simple JavaScript-like language. Test your backend with API monitors, which allow you to run scripted tests against any API endpoint.
+Scripted browsers expand your testing capabilities, so you can test uncommon user flows or beta-test complex procedures. For example, ensure your users are able to sign up for your newsletter, add an item to their cart, or search for and find a piece of critical content with a simple JavaScript-like language. Test your backend with API monitors, which allow you to run scripted tests against any API endpoint.
```
Comment (jeff-colucci) [inline src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx:223]: In the second sentence, would you consider adding "you can" before "ensure"? I only bring it up because when I read through it I'm expecting a procedure to follow the "ensure" part (ie: ensure your users are able to sign up... by"), but then it kind of turns into a list of things you can do instead. Again, just a small thing that's not gating.

**Review by jeff-colucci — APPROVED** [review]
Only a few small things that I found but overall looks great! I don't think any of the stuff I found is gating so I will go ahead and leave any changes you want to make up to you and approve.
